### PR TITLE
Issue #99 -- Should Submission Requirements take precedence over Input Descriptors?

### PR DESCRIPTION
**Pull request by glcohen**
Address #99

### CODE REVIEW

**Comment by glcohen** [comment]
@brentzundel @csuwildcat updated the language

File: spec/spec.md [hunk 1]
```diff
@@ -539,7 +539,7 @@ party has asserted (via a `Proof Submission` object). The following section
 defines the format for _Submission Requirement_ objects and the selection syntax
 verifying parties can use to specify which combinations of inputs are acceptable.
 
-All members of the `submission_requirements` array ****MUST**** be satisfied. If present, `input_descriptors`, not satisfied by `submission_requirements`, ****MUST**** be satisfied.
+All members of the `submission_requirements` array ****MUST**** be satisfied. All `input_descriptors` must be grouped. There may be `input_descriptors` left unfulfilled after satisfying all `submission_requirements`.
```
Comment (csuwildcat) [inline spec/spec.md:542]: Consider the following language:

If present, all members of the `submission_requirements` array MUST be satisfied, and all `input_descriptors` MUST be grouped. Any unused `input_descriptors` that remain after satisfying all `submission_requirements` MUST be ignored.

Comment (glcohen) [reply spec/spec.md:542]: updated

File: index.html [hunk 2]
```diff
@@ -803,10 +803,10 @@ <h4 id="property-values-and-evaluation"><a class="toc-anchor" href="#property-va
 </ol>
 <h3 id="input-descriptors"><a class="toc-anchor" href="#input-descriptors">§</a> Input Descriptors</h3>
 <p><em>Input Descriptors</em> are objects used to describe the information a Verifier
-requires of a Holder before they will proceed with an interaction. <em>Input
-Descriptor Objects</em> contain a schema URI that links to the schema of the
-required input data, constraints on data values, and an explanation why a
-certain item or set of data is being requested:</p>
+requires of a Holder before they will proceed with an interaction. If no <code>submission_requirement</code> objects are presenet, to successfully fulfill a <code>Presentation Definition</code>, all <code>input_descriptor</code> objects <strong><strong>MUST</strong></strong> be satisfied.</p>
```
Comment (brentzundel) [inline index.html:806]:
```suggestion
requires of a Holder before they will proceed with an interaction. If no <code>submission_requirement</code> objects are present, all <code>input_descriptor</code> objects <strong><strong>MUST</strong></strong> be satisfied.</p>
```
I think this says about the same thing, but reads a bit clearer to me.

Comment (glcohen) [reply index.html:806]: much nicer! updated.

**Review by csuwildcat — APPROVED** [review]

**Pull request merged**